app/views/includes/scripts_ga_1.html

```diff
@@ -1,11 +1,9 @@
 {% if globals.enableTracking == "true" %}
-    <!-- Google Tag Manager -->
-    <script nonce="{{ globals.nonce }}">
-        (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
+<!-- Google Tag Manager -->
+<script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
         new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
-        j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
-        'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
-        })(window,document,'script','dataLayer','GTM-KG8XT3S');
-    </script>
-    <!-- End Google Tag Manager -->
+    j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
+    'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
+})(window,document,'script','dataLayer','GTM-KG8XT3S');</script>
+<!-- End Google Tag Manager -->
 {% endif %}
```

Comment:
Undo all of this.

Reply:
done

Reply:
Did you push? I still see these changes.